docs/event-bus/docs/040-details-troubleshooting.md

```diff
@@ -21,6 +21,21 @@ document introduces several ways to troubleshoot such problems.
  If these general guidelines do not help, go to the next section of this
  document.
 
+## Search by tags
+It is possible to search the traces using tags. Tags are “key-value” pairs configured for each service.
+
+The full list of tags for a service can be seen from the details of that service's span.
+
+e.g. Here are the spans for the `publish-service`.
+* `event-type`
+* `event-type-ver`
+* `event-id`
+* `source-id`
+
+To search, one can either use 
```

Comment:
``To search the traces, you can use either a single tag such as  `event-type="order.created"`, or multiple tags such as  `event-type="order.created" event-type-ver="v1"`.``

No need to introduce a list for 2 items only. My question is: are  `event-type="order.created"` and `event-type="order.created" event-type-ver="v1"` the examples of signle and multiple tags?